fennel/sources/test_sources.py

```diff
@@ -1247,3 +1247,148 @@ class UserInfoDataset:
         assert (
             suffix == expected_suffix
         ), f"Expected suffix: {expected_suffix}, got: {suffix}"
```

Comment:
can you assert on an unbounded source as well that bounded is set to False and idleness is set to None?